src/components/Pages/HomePage/HomePage.js

```diff
@@ -79,8 +79,11 @@ class HomePage extends React.Component {
     const { __is_custom_site, community } = this.props;
     const { subdomain } = community || {};
 
-    const prefix = !__is_custom_site && subdomain ? `/${subdomain}` : "";
-    
+    const { hostname } = new URL(window.location.href);
+    const isNotCustom = URLS.NONE_CUSTOM_WEBSITE_LIST.has(hostname);
+
+    const prefix = isNotCustom && subdomain ? `/${subdomain}` : "";
```

Comment:
do you mean suffix?
```suggestion
    const suffix = isNotCustom && subdomain ? `/${subdomain}` : "";
```

Reply:
No, I meant prefix. In concatenation, we add this part, before the actual link the user provides...  Why suffix?